molecule/rolling_update/tests/test_dists_rotated.py

```diff
@@ -1,10 +1,24 @@
 import re
+import sys
+
+from testinfra.modules.file import File
 
 import utils
 
 testinfra_hosts = utils.get_testinfra_hosts()
 
 
+def listdir(self):
+    out = self.run_test("ls -1 -q -- %s", self.path)
```

Comment:
Wasn't mentioned in description, I didn't understand.

Reply:
This function doesn't exists in Python 2, so it's a copy-paste from Python 3 library